test/yaml-rest-runner/src/main/java/org/elasticsearch/test/rest/yaml/section/SkipSection.java

```diff
@@ -172,13 +208,17 @@ static SkipSectionBuilder parseInternal(XContentParser parser) throws IOExceptio
                 currentFieldName = parser.currentName();
             } else if (token.isValue()) {
                 if ("version".equals(currentFieldName)) {
-                    builder.withVersion(parser.text());
+                    builder.skipVersion(parser.text());
                 } else if ("reason".equals(currentFieldName)) {
-                    builder.withReason(parser.text());
+                    builder.setReason(parser.text());
                 } else if ("features".equals(currentFieldName)) {
                     parseFeature(parser.text(), builder);
                 } else if ("os".equals(currentFieldName)) {
-                    builder.withOs(parser.text());
+                    builder.skipOs(parser.text());
+                } else if ("cluster_has_features".equals(currentFieldName)) {
+                    builder.skipIfClusterFeaturesPresent(parser.text());
+                } else if ("cluster_lacks_features".equals(currentFieldName)) {
```

Comment:
Does it make sense to use the same terminology here? `cluster_features_present`, `cluster_features_absent`?

Reply:
I agree - I was waiting to see which way we want to go to rename them, but better do it now an keep things consistent 👍